### PR TITLE
Add `Irregular layout` example

### PR DESCRIPTION
**Pull request by wwdc14**
Preview

### CODE REVIEW

**Comment by wwdc14** [comment]
How to fix this error?

**Comment by lkzhao** [comment]
Can you rebase your changes?

File: Examples/UIComponentExample/Examples/Irregular Layout/Components/CustomizeLayout.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+//  UIComponentExample
+//
+//  Created by y H on 2021/7/30.
+//  Copyright © 2021 Luke Zhao. All rights reserved.
+//
+
+import UIKit
+import UIComponent
+
+struct CustomizeLayout: Component, HorizontalLayoutProtocol {
```
Comment (lkzhao) [inline Examples/UIComponentExample/Examples/Irregular Layout/Components/CustomizeLayout.swift:12]: There is a `Absolute` layout builtin.

Comment (wwdc14) [reply Examples/UIComponentExample/Examples/Irregular Layout/Components/CustomizeLayout.swift:12]: `Absolute`  cannot get the `constraint` parameter
Need `constraint` calc reasonable size
```swift
VStack(spacing: 2) {
  Absolute(children: [], frames: []) // cannot get the constraint parameter
}.inset(h: 10)
```

**Comment by wwdc14** [comment]
check #13

**Pull request closed without merging**